--- authconfig-6.2.2/Makefile.am~	2012-03-27 13:16:24.000000000 +0300
+++ authconfig-6.2.2/Makefile.am	2012-04-20 09:33:15.148056514 +0300
@@ -84,6 +84,7 @@
 
 acutilmodule_la_SOURCES = acutilmodule.c
 acutilmodule_la_LDFLAGS = -module -avoid-version -export-dynamic @LDFLAGS@
+acutilmodule_la_LIBS = -lresolv
 
 EXTRA_DIST = authconfig.spec.in authconfig.spec authconfig.pamd \
 	authconfig.console authconfig-tui.console authconfig-gtk.console \
